### ISSUE TITLE
Which assembly is it?

### ISSUE THREAD

**Issue by dr43058** [manunt/AssemblyVisualizer#7]
What is the name of the assembly for AssemblyVisualizer I am supposed to put in the directory for Reflector c:\Prog Files(x86)\Red Gate\.NET Reflector\Desktop 8.4. There is a folder called AddIns, is that it?

**Comment by dr43058** [comment]
![assemblyvisualizer](https://cloud.githubusercontent.com/assets/426780/6337269/53731d2a-bb6e-11e4-86d0-e7ba10f0c0e6.PNG)

**Comment by manunt** [comment]
Assembly Visualizer plugin is just a single assembly with the name AssemblyVisualizer.Plugin.dll. Builds for ILSpy and Reflector are not compatible, [here](https://github.com/downloads/manunt/AssemblyVisualizer/Reflector%20Visualizer%201.0.2.0.zip) you can get assembly for Reflector.

**Comment by dr43058** [comment]
Okay, I got the dll you provided in the link. But, now I am getting an error from the "Add-Ins" windows. The error says...
System.IO.FileException: Could not load file or assembly 'AssemblyVisualizer.Plugin.dll' or one of it's dependencies. Access is denied. 
File name: 'AssemblyVisualizer.Plugin.dll'
at System.Reflection.RuntimeAssembly._ .......elided

I checked windows permissions and it mostly matched what the powercommands.dll had
